zoo-css-challenge/style.css

```diff
@@ -8,7 +8,9 @@ header {
   width: 100%;
   padding: 20px;
 }
-
+#logo{
+  border:none;
+}
```

Comment:
```suggestion
```
Instead of adding `border:none;`, we could instead delete the border CSS below

```diff
-img {
-  border: 5px solid #8075ff;
-}
```